internal/pkg/cli/app_init.go

```diff
@@ -232,7 +238,22 @@ func (o *initAppOpts) validateAppName(name string) error {
 	if err != nil {
 		var noSuchAppErr *config.ErrNoSuchApplication
 		if errors.As(err, &noSuchAppErr) {
-			return nil
+			roleName := fmt.Sprintf("%s-adminrole", name)
```

Comment:
since there's a lot of logic in the `if err != nil` case, i wonder if it makes more sense to handle the `err == nil` case _first_ to reduce the amount of nested code here 🤔